<ui version="4.0" >
 <class>YaRosterTipLabel</class>
 <widget class="QWidget" name="YaRosterTipLabel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>11</number>
   </property>
   <property name="rightMargin" >
    <number>10</number>
   </property>
   <property name="bottomMargin" >
    <number>10</number>
   </property>
   <property name="horizontalSpacing" >
    <number>11</number>
   </property>
   <property name="verticalSpacing" >
    <number>7</number>
   </property>
   <item row="1" column="0" colspan="2" >
    <widget class="PsiTextView" name="statusTypeDescriptionLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <property name="text" stdset="0" >
      <string>Current status</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="minimumSize" >
      <size>
       <width>101</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>101</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>0</number>
      </property>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="YaContactAvatarLabel" name="contactUserpic" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>100</width>
          <height>100</height>
         </size>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>32</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="YaPushButton" name="chatButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Chat</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/chat_send.png</normaloff>:/images/chat_send.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="historyButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>History</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_history.png</normaloff>:/images/user-actions/button_glyph_history.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="profileButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Profile</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_profile.png</normaloff>:/images/user-actions/button_glyph_profile.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="photosButton" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Photos</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_fotki.png</normaloff>:/images/user-actions/button_glyph_fotki.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>16</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>7</number>
     </property>
     <item>
      <widget class="QLabel" name="jidIconLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>14</width>
         <height>14</height>
        </size>
       </property>
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaJidLabel" name="jidLabel" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="frameShape" >
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text" >
        <string>user@ya.ru</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>100</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>9</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QLabel" name="genderLabel" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize" >
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>12</width>
            <height>12</height>
           </size>
          </property>
          <property name="text" >
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>5</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="FadingMultilineLabel" name="infoLabel" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text" >
         <string>66 anni</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="1" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>0</number>
     </property>
     <item>
      <widget class="YaPushButton" name="renameButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Rename</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_rename.png</normaloff>:/images/user-actions/button_glyph_rename.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="addButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_add.png</normaloff>:/images/user-actions/button_glyph_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="authButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Auth</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="deleteButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Delete</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_remove.png</normaloff>:/images/user-actions/button_glyph_remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="YaPushButton" name="blockButton" >
       <property name="focusPolicy" >
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text" >
        <string>Block</string>
       </property>
       <property name="icon" >
        <iconset resource="yastuff.qrc" >
         <normaloff>:/images/user-actions/button_glyph_block.png</normaloff>:/images/user-actions/button_glyph_block.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="YaContactLabel" name="contactName" >
     <property name="text" >
      <string>Mario Rossi</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>YaPushButton</class>
   <extends>QPushButton</extends>
   <header>yapushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>YaJidLabel</class>
   <extends>QLabel</extends>
   <header>yajidlabel.h</header>
  </customwidget>
  <customwidget>
   <class>YaContactAvatarLabel</class>
   <extends>QFrame</extends>
   <header>yacontactavatarlabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PsiTextView</class>
   <extends>QTextEdit</extends>
   <header>psitextview.h</header>
  </customwidget>
  <customwidget>
   <class>YaContactLabel</class>
   <extends>QLabel</extends>
   <header>yacontactlabel.h</header>
  </customwidget>
  <customwidget>
    <class>FadingMultilineLabel</class>
    <extends>QLabel</extends>
    <header>fadingmultilinelabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="yastuff.qrc" />
 </resources>
 <connections/>
</ui>
